types/temperature.yaml

```diff
@@ -0,0 +1,13 @@
+description: Provides temperature readings from all 6 sensors we have
+types:
+  Temperatures:
+    description: Temperatures read from MCU
+    type: object
+    properties:
```

Comment:
Everest has no concept of namespaces, so "temperature:" would conflict with every other use of temperature. So I would suggest to either
* drop the message and use generic array (the implementation id already says temperature)
* rename it to phyverso_mcu_temperature
* create a "generic" temperature (maybe tuples of temperature in kelvin and sensor name)

Reply:
@wku12 Thats not what I meant - I mean rename the type and and the interface. Otherwise every user of temperature messages will have this field